go.mod

```diff
@@ -18,6 +18,7 @@ require (
 )
 
 require (
+	github.com/aws/aws-sdk-go v1.44.131 // indirect
```

Comment:
Don't think we need to pull in v1 of the SDK here.

Reply:
oh idk what happened? it's indirect so must be something's dependency.